internal/pkg/manifest/job.go

```diff
@@ -145,6 +146,11 @@ func (j *ScheduledJob) BuildRequired() (bool, error) {
 	return requiresBuild(j.ImageConfig.Image)
 }
 
+// PublishArgs returns a manifest.PublishConfig object for the service
+func (s *ScheduledJob) PublishCfg() *PublishConfig {
```

Comment:
I think by pushing the ARN computation to the `stack` or `template` pkg we can remove this method